front/pages/api/w/[wId]/assistant/conversations/[cId]/participants.ts

```diff
@@ -67,12 +68,66 @@ async function handler(
       res.status(200).json({ participants: participantsRes.value });
       break;
 
+    case "POST":
+      const user = auth.user();
+      if (!user) {
+        return apiError(req, res, {
+          status_code: 401,
+          api_error: {
+            type: "app_auth_error",
+            message: "User not authenticated",
+          },
+        });
+      }
+
+      const owner = auth.workspace();
+      if (!owner) {
+        return apiError(req, res, {
+          status_code: 401,
+          api_error: {
+            type: "app_auth_error",
+            message: "Workspace not found",
+          },
+        });
+      }
+
+      const existingParticipant = await ConversationParticipantModel.findOne({
+        where: {
+          conversationId: conversationWithoutContent.id,
+          workspaceId: owner.id,
+          userId: user.id,
+        },
+      });
+
+      if (existingParticipant) {
+        return apiError(req, res, {
+          status_code: 400,
+          api_error: {
+            type: "invalid_request_error",
```

Comment:
here, changing the invalid_request_error to a user_already_participant or smth similar